packages/example-codehub/src/codehub-application.ts

```diff
@@ -25,13 +28,13 @@ export class CodeHubApplication extends Application {
     app.controller(UserController);
     app.controller(HealthController);
 
-    app.bind('userId').to(42);
+    server.bind('userId').to(42);
 
-    app.bind('app.info').toDynamicValue(() => this.info());
+    server.bind('app.info').toDynamicValue(() => this.info());
```

Comment:
Ditto, I think `app.info` should be bound on the application. However, it makes sense to calculate the returned information by gathering info from `Server` instances.